build.gradle

```diff
@@ -236,29 +236,33 @@ allprojects {
         google()
         mavenCentral()
         // org.jetbrains.trove4j:trove4j:20160824.
-        maven { url 'https://plugins.gradle.org/m2/' }
+        maven {
+            url 'https://plugins.gradle.org/m2/'
+        }
         maven {
             url 'https://trouterpublicpackages.z13.web.core.windows.net'
         }
     }
 
     ext {
+        androidxTestVersion = "1.3.0"
         annotationsVersion = "1.1.0"
         appCompatVersion = "1.2.0"
         archCoreTestVersion = "2.1.0"
+        azureCoreVersion = "1.0.0-beta.4"
         concurrentFuturesVersion = "1.0.0"
-        jUnitVersion = "5.7.0"
-        androidxTestVersion = "1.3.0"
+        dataproviderVersion =  "2.6"
         jacksonVersion = "2.11.2"
+        jUnitVersion = "5.7.0"
         lifecycleVersion = "2.2.0"
+        nimbusJoseJwtVersion = "9.1.2"
         okHttpVersion = "3.14.9"
         retrofitVersion = "2.9.0"
         roomVersion = "2.2.5"
         staxApiVersion = "1.0-2"
-        threeTenAbpVersion = "1.2.4"
-        threeTenBpVersion = "1.4.4"
+        threeTenAbpVersion = "1.3.0"
+        threeTenBpVersion = "1.5.0"
```

Comment:
Just make sure this minor version update will not affect our targeted min version 15.

Reply:
ThreeTenABP's `minSdkVersion` has not changed since it's original commit ([it's `15`](https://github.com/JakeWharton/ThreeTenABP/blob/25d21edcf8703a38fabdfa17712352af6feb4a07/build.gradle#L31)) and [it uses ThreeTenBp `1.5.0`](https://github.com/JakeWharton/ThreeTenABP/blob/25d21edcf8703a38fabdfa17712352af6feb4a07/build.gradle#L40), so we should be in the clear.